pkg/apis/acme/v1alpha2/types_issuer.go

```diff
@@ -355,8 +355,7 @@ type ACMEIssuerDNS01ProviderAcmeDNS struct {
 // ACMEIssuerDNS01ProviderRFC2136 is a structure containing the
 // configuration for RFC2136 DNS
 type ACMEIssuerDNS01ProviderRFC2136 struct {
-	// The IP address of the DNS supporting RFC2136. Required.
-	// Note: FQDN is not a valid value, only IP.
+	// The IP address or hostname of an authoritative DNS server supporting RFC2136. Required.
 	Nameserver string `json:"nameserver"`
```

Comment:
Can you make a similar corresponding change in `pkg/apis/acme/v1alpha3/types_issuer.go` and `pkg/internal/apis/acme/types_issuer.go`? 😄